distribution/docker/src/docker/Dockerfile

```diff
@@ -3,45 +3,53 @@
 #
 # Beginning of multi stage Dockerfile
 ################################################################################
+<% /*
```

Comment:
Because this is in a code block, this is _not_ included in the final published file, right?

Reply:
Exactly. I also escaped all the backslashes in the file, so that the final rendered Dockerfile is much improved.